nixos/modules/programs/preload.nix

```diff
@@ -0,0 +1,215 @@
+{ config, lib, pkgs, runCommand, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.preload;
+  configFile = pkgs.writeText "preload.conf"
+    ''
+      [model]
+      cycle = ${cfg.cycle}
+      usecorrelation = ${cfg.usecorrelation}
+      minsize = ${cfg.minsize}
+      memtotal = ${cfg.memtotal}
+      memfree = ${cfg.memfree}
+      memcached = ${cfg.memcached}
+
+      [system]
+      doscan = ${cfg.doscan}
+      dopredict = ${cfg.dopredict}
+      autosave = ${cfg.autosave}
+      mapprefix = ${cfg.mapprefix}
+      exeprefix = ${cfg.exeprefix}
+      processes = ${cfg.processes}
+      sortstrategy = ${cfg.sortstrategy}
+    '';
```

Comment:
Please use rfc42 style options. then we wont have to have so many separate options.

Reply:
Fixed on latest commits